content/guides/tooling/sdks/sdk-versioning.md

```diff
@@ -13,63 +13,90 @@ alias_paths: []
 
-## How it will work
+## What does it mean for your project
 
-Box core SDKs and Box Next Generation SDKs were created as separate libraries. Going forward, the Box core SDK artifacts will include:
+<TileGrid rows="2">
+  <Tile type="cog" title="1. Existing App - Box core SDK" href="/guides/tooling/sdks/sdk-versioning/#migration-to-the-generated-package">
+    If you have an existing application relying on Box core SDK and you wish to further develop your project:
 
-- `(n+1)` major version will include manually maintained **Box core SDK** and **Box Next Generation SDK** artifacts. This version enables leveraging coexisting artifacts at the same time, and serve as a transition phase. Stay tuned for this upcoming release.
-- `v10.0.0` version includes **only the Box Next Generation SDK artifact**. Currently, this version is available as a branch for each SDK repository.
+    **Action** Use the major version which includes consolidated packages. Start leveraging convenience methods, new features thanks to the generated package, and gradually migrate your code base. Check version breakdown for each programming language.
+  </Tile>
+  <Tile type="code-new" title="2. Existing App - Next Generation SDK" href="/guides/tooling/sdks/sdk-versioning/#migrating-from-box-next-generation-sdks-to-core-box-sdk-v10">
+    If you have an existing application relying on the Box Next Generation SDK and you wish to further develop your project:
 
-## How to decide
+    **Action** Replace the library name in your package manager with Box core SDK package (≥`v10.0.0`). Check migration guides for detailed instructions.
+  </Tile>
+  <Tile type="branch" title="3. New Application" href="/guides/tooling/sdks/sdk-versioning/#box-core-sdk-versions-and-artifacts-overview">
+    If you are creating a new application:
 
-1. If you are creating a new application, use `v10` of the Box core SDK package.
-2. If you have an existing application relying on the **Box Next Generation SDK** and you wish to further develop your project, replace the name of this library in the package manager with the Box core SDK package (`v10.0.0`). Object imports for most SDKs will persist the same and your code will work as-is; the TypeScript SDK requires additional steps for migrating. Check the section below for detailed guides.
-3. If you have an existing application relying on **Box core SDK** and you wish to further develop your project, once released, bump the library version by one major release. See the table above for a detailed breakdown of recommended versions.
-4. If you have an existing application that you don’t plan to change, ensure your package manager includes the version of SDK you are using to prevent an accidental rebuild that pulls in a version you aren’t expecting.
+    **Action** Use ≥`v10.0.0` of the Box core SDK package.
+  </Tile>
+  <Tile type="info" title="4. No Active Development" href="/guides/tooling/sdks/sdk-versioning/#box-core-sdk-versions-and-artifacts-overview">
```

Comment:
I would add one more case:

App using both Box Next Generation SDK and old sdk -> use combined SDK

Reply:
Done